### ISSUE TITLE
Problem. radio() with value 0 display label

### ISSUE THREAD

**Issue by seedo** [dwightwatson/bootstrap-form#68]
hi
`BootForm::radio('node[check]', 'Check', 0)`
render
`<label for="node[check]"><inpu name="node[check]" id="node[check]" type="radio" value="Check">Check</label>`
maybe change `$value = $value ?: $label;` to`$value = is_null($value) ? $label : $value;` in
`public function radioElement($name, $label = null, $value = null, $checked = null, $inline = false, array $options = [])`
it will happen when use BootForm::radios() with values ['0'=>'No', '1'=>'Yes']
Thanks you.

**Comment by dwightwatson** [comment]
Those two lines of code should do the same thing. If `value` is present use it, otherwise use `label`. Is this right or am I missing something?

**Comment by seedo** [comment]
Thanks for reply.
When $value is 0 or '', `$value ?: $label` will return $lable not $value.
`{!! BootForm::radios('node[status]', 'Published', ['0'=>'No', '1'=>'Yes'], $node->status, true) !!}` render
```
<label class="radio-inline"><inpu name="node[status]" type="radio" value="**No**">No</label>
<label class="radio-inline"><inpu name="node[status]" type="radio" value="1">Yes</label>
```
First $value is not I expected '0' but 'No', maybe I use wrong way?

**Comment by hodaEdrob** [comment]
Hi 

This has caused a lot of problems with my forms that all use` ['No', 'Yes']` array as the radios choices, it thinks value 0 equals to having no value at all and sets the value of 'No' to it's label 'No'. All my code expects No to be zero not "No" please can you check for zero before setting the value to it's label?

It didnt use to do this before.

**Comment by dwightwatson** [comment]
Happy to accept a PR that fixes this. I'm not sure at what point this was broken.

**Comment by hodaEdrob** [comment]
Hi 

Thanks a lot for the reply, I have made a pull request, if you could double check that it's ok, it seems to work for me. So it sets the value for "No" to zero and for "Yes" to 1 if I pass `['No', 'Yes']` to it.

**Comment by dwightwatson** [comment]
I've merged #71 into `master` which should fix this issue (thanks @hodaEdrob).

Please test if you can, and I'll tag a fresh versioned release tomorrow.

**Comment by dwightwatson** [comment]
Thanks everyone for your patience, I've just tagged a new release - `1.0.19`.
